src/components/books/Bookops.jsx

```diff
@@ -1,34 +1,33 @@
 /* eslint-disable */
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../../redux/books/booksSlice';
 
-const Bookops = ({ handleOperations }) => (
-  <div className="book-actions">
-    <button
-      type="button"
-      onClick={() => handleOperations(0)}
-      className="btn btn-comment"
-    >
-      Commnet
-    </button>
-    <button
-      type="button"
-      onClick={() => handleOperations(0)}
-      className="btn btn-remove"
-    >
-      Remove
-    </button>
-    <button
-      type="button"
-      onClick={() => handleOperations(0)}
-      className="btn btn-edit"
-    >
-      Edit
-    </button>
-  </div>
-);
+const Bookops = ({ bookId }) => {
+  const dispatch = useDispatch();
+  console.log('bookId', bookId);
```

Comment:
- Please remove all `console.log` statements, it would help make your project more professional, please fix this 😄